modules/videoio/test/test_ffmpeg.cpp

```diff
@@ -476,6 +476,23 @@ static void ffmpeg_check_read_raw(VideoCapture& cap)
     EXPECT_EQ((size_t)37118, data.total());
 }
 
+TEST(videoio_ffmpeg, ffmpeg_check_extra_data)
+{
+    if (!videoio_registry::hasBackend(CAP_FFMPEG))
+        throw SkipTestException("FFmpeg backend was not found");
+
+    string video_file = findDataFile("video/big_buck_bunny.mp4");
+    VideoCapture cap;
+    EXPECT_NO_THROW(cap.open(video_file, CAP_FFMPEG));
+    Mat data;
+    int codecExtradataIdx = (int)cap.get(CAP_PROP_CODEC_EXTRADATA_INDEX);
+    EXPECT_TRUE(codecExtradataIdx > 0);
```

Comment:
> EXPECT_TRUE(codecExtradataIdx > 0);

FFmpeg wrapper for Windows is updated separately, so test code should be aligned.

```
#ifdef _WIN32  // handle old FFmpeg backend
if (codecExtradataIdx <= 0)
    throw SkipTestException("Codec extra data is not supported");`
#endif
```

Reply:
👍 thumb